### ISSUE TITLE
Firebase doesn't support Cocoapods 1.7.0 with Multiple Xcodeproj Generation feature

### ISSUE THREAD

**Issue by smolskyaleksey** [firebase/firebase-ios-sdk#3109]
Add to podfile line
```
install! 'cocoapods', :generate_multiple_pod_projects => true
```
and build again

I get error 
``` swift
'FIRApp.h' file not found
```

**Comment by paulb777** [comment]
Please update to the latest version.  Support for `:generate_multiple_pod_projects`  was added in #2751 and released with Firebase 6.0.0.

**Comment by smolskyaleksey** [comment]
thx a lot
